### PR TITLE
docs(guides): adds I/O codelab to list of codelabs

### PR DESCRIPTION
**Pull request by skim1420**
@demobox @lwander fyi. Not sure

### CODE REVIEW

File: guides/tutorials/codelabs/index.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Run through these codelabs to get hands-on experience with Spinnaker in a guided
 
 ## Table of Contents
 
+* [Continuous Delivery to Kubernetse Using Spinnaker (Google I/O 2018 Spinnaker Codelab)](https://codelabs.developers.google.com/codelabs/cloud-spinnaker-kubernetes-cd/index.html?index&authuser=0#0) - deploy an application from Kubernetes manifest files
```
Comment (demobox) [inline guides/tutorials/codelabs/index.md]: [minor] Curious about the `?index&authuser=0#0` part of the URL - is that needed?

Also, from a brief glance, Google I/O 2018 isn't obviously mentioned - perhaps call it "Kubernetes manifest sample pipelines" or so?

/cc @lwander

Comment (skim1420) [reply guides/tutorials/codelabs/index.md]: Yep, cleaned up the URL, thanks. Also the link description does have Google I/O 2018 in it. Is that what you mean?

Comment (demobox) [reply guides/tutorials/codelabs/index.md]: > Also the link description does have Google I/O 2018 in it. Is that what you mean?

I meant the "Google I/O 2018 Spinnaker Codelab" description, yes.

File: guides/tutorials/codelabs/index.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ Run through these codelabs to get hands-on experience with Spinnaker in a guided
 
 ## Table of Contents
 
+* [Continuous Delivery to Kubernetse Using Spinnaker (Google I/O 2018 Spinnaker Codelab)](https://codelabs.developers.google.com/codelabs/cloud-spinnaker-kubernetes-cd/) - deploy an application from Kubernetes manifest files
```
Comment (duftler) [inline guides/tutorials/codelabs/index.md]: s/Kubernetse/Kubernetes

Comment (skim1420) [reply guides/tutorials/codelabs/index.md]: fixed

File: guides/tutorials/codelabs/index.md (hunk 1 above)
Comment (skim1420) [reply guides/tutorials/codelabs/index.md]: Done.

**Pull request merged**